src/System.Net.Security/src/System/Net/SslStreamPal.Unix.cs

```diff
@@ -70,55 +70,17 @@ public static SafeFreeContextBufferChannelBinding QueryContextChannelBinding(Saf
             throw NotImplemented.ByDesignWithMessage(SR.net_MethodNotImplementedException);
         }
 
-        public static int QueryContextStreamSizes(SafeDeleteContext securityContext, out StreamSizes streamSizes)
+        public static void QueryContextStreamSizes(SafeDeleteContext securityContext, out StreamSizes streamSizes)
         {
             streamSizes = null;
-            try
-            {
-                streamSizes = new StreamSizes(Interop.libssl.SslSizes.HEADER_SIZE, Interop.libssl.SslSizes.TRAILER_SIZE, Interop.libssl.SslSizes.SSL3_RT_MAX_PLAIN_LENGTH);
-                return 0;
-            }
-            catch
-            {              
-                return -1;
-            }
+            streamSizes = new StreamSizes(Interop.libssl.SslSizes.HEADER_SIZE, Interop.libssl.SslSizes.TRAILER_SIZE, Interop.libssl.SslSizes.SSL3_RT_MAX_PLAIN_LENGTH);
         }
 
-        public static int QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
+        public static void QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
         {
             connectionInfo = null;
```

Comment:
Sorry about the delay: the code changed since this PR: https://github.com/dotnet/corefx/pull/3525/files

Regarding this particular code: `connectionInfo` is an `out` parameter and it needs to be populated within the method.